setup.py

```diff
@@ -1,23 +1,50 @@
+# this restructured text docstring is displayed on https://pypi.python.org/pypi/notifications-python-client/
 """
-Python API client for GOV.UK Notify
+Python API client for GOV.UK Notify - see https://www.notifications.service.gov.uk/ for more information.
+
+For usage and documentation see https://github.com/alphagov/notifications-python-client.
 """
 import re
 import ast
 import pip.download
 from pip.req import parse_requirements
 from setuptools import setup, find_packages
+from setuptools.command.test import test as TestCommand
 
-
+# can't just import notifications_python_client.version as requirements may not be installed yet and imports will fail
 _version_re = re.compile(r'__version__\s+=\s+(.*)')
 
 with open('notifications_python_client/version.py', 'rb') as f:
     version = str(ast.literal_eval(_version_re.search(
         f.read().decode('utf-8')).group(1)))
 
-requirements = list(parse_requirements('requirements.txt',
-                                       session=pip.download.PipSession()))
 
-install_requires = [str(r.req) for r in requirements]
+def get_reqs(requirement_file):
+    requirements = list(parse_requirements(requirement_file, session=pip.download.PipSession()))
+    return [str(r.req) for r in requirements]
+
+
+install_requires = get_reqs('requirements.txt')
+tests_require = get_reqs('requirements_for_test.txt')
+# for running pytest as `python setup.py test`, see
+# http://doc.pytest.org/en/latest/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner
+setup_requires = ['pytest-runner']
+
+
+class IntegrationTestCommand(TestCommand):
```

Comment:
possibly. all the examples i saw have it in setup.py ( and not just tutorials which might be kept together for brevity, eg https://github.com/pytest-dev/pytest/blob/master/setup.py#L86, https://github.com/zzzeek/sqlalchemy/blob/master/setup.py#L77), because it makes no sense outside of the setup process. it's not part of the actual usage of the lib or its surrounding helpers if that makes sense - all it's doing is calling `py.test integration_tests/integration_test_suite.py` anyway